pluginManager/projects/2015/PluginManager.vcxproj.filters

```diff
@@ -123,7 +123,6 @@
     </ClCompile>
   </ItemGroup>
   <ItemGroup>
-    <Image Include="..\..\res\nbc_logo.png" />
     <Image Include="..\..\res\nbc_logo.bmp">

```

Comment:
Just the bmp is available in git, is this intended? VS automatically removes the unkown image file from the filters.